motion/project/vendor.rb

```diff
@@ -185,7 +185,7 @@ def build_xcode(platform)
       end
 
       build_dir = build_dir(platform)
-      if !File.exist?(build_dir) or Dir.glob('**/*').sort.any? { |x| File.mtime(x) > File.mtime(build_dir) }
+      if !File.exist?(build_dir) or Dir.glob('**/*').sort.any? { |x| File.exists?(x) && File.mtime(x) > File.mtime(build_dir) }
```

Comment:
Did you mean to use `File.exist?` as opposed to `File.exists?` here?

Reply:
Yes I meant to use `File.exist?` - I always forget `exists?` is deprecated.  The second commit I pushed fixed it.